### DISCUSSION TITLE
Anti Ice Alarm

### DISCUSSION THREAD

**Discussion by Why5oSerious** [flybywiresim/aircraft#2400] [Help] [answered]
Is it typical to get an Anti Ice Alarm, by clear sky and SAT-64°C? thought only between +10°C and -40°C and clowdy, foggy or wet conditions.

![Download](https://user-images.githubusercontent.com/75929032/102007319-4032f680-3d28-11eb-934e-cddfa7d4f9aa.png)

**Comment by beheh** [comment, marked as answer]
So the icing is unfortunately controlled by the simulator. We just read the current icing values and alert you if they're too high.

**Reply by Why5oSerious** [reply]
ok thx. Good to know.

**Comment by lve0200** [comment]
Does the msg mean that the Anti Ice is switched automatically ON, although the OVH panel buttons are not lit? Should they??

**Reply by beheh** [reply]
No, they're not. You should turn them on automatically. The cyan lines are instructions, they go away once you've performed them.
